third_party/payjs/README.amp

```diff
@@ -0,0 +1,9 @@
+URL: https://github.com/google/payjs
+License: Apache 2.0
+License File: https://github.com/google/payjs/blob/master/LICENSE
+
+Description:
+Copy of Google Pay integration library
```

Comment:
@zwd199032 Any plans to publish this on NPM? It would make upgrades much easier on amphtml.

Reply:
No plan as for now. We'll probably evaluate whether we want to do this if we decide to frequently update this package. We can always put a npm dependency to replace this if we go toward that direction in the future.